ports/ignition-cmake0/portfile.cmake

```
@@ -7,7 +7,7 @@ set(PACKAGE_VERSION "0.6.1")
 ignition_modular_library(NAME cmake
                          VERSION ${PACKAGE_VERSION} 
                          REF "ignition-cmake_${PACKAGE_VERSION}"
-                         SHA512 fcd3ad6b5289697c4928c71b820e2adaa758c730f52cba3f8cc714e44ca0c9f04f432ae5b98b5f258c4851c4666740b58066a25c55ff3a6de975cd8a57991b6b
+                         SHA512 bd57cd43dd944fef264353b67fbbbab989d4bb638b684f80868a8b61fe0b854e156e00852b967c7caa8598315bea60bd8b27ff000410e93c4f04185a13c90489

```

CI seems to be complaining about this hash:

```suggestion
                         SHA512 fcd3ad6b5289697c4928c71b820e2adaa758c730f52cba3f8cc714e44ca0c9f04f432ae5b98b5f258c4851c4666740b58066a25c55ff3a6de975cd8a57991b6b

```

---

I am relatively sure  that new one is correct (I tested locally several times), and the one you suggested (that I guess you got from the CI) is actually the old one (the one  of the archive generated by bitbucket). Perhaps the archive  have the same names, and so there is some caching happening in the `downloads` folder in the CI so that the new GitHub archive is not actually downloaded, but instead the CI tries to use the old Bitbucket archive?

---

I  actually checked and indeed the archive created by `vcpkg_from_bitbucket` and `vcpkg_from_github`, so also a user would get the same error by: 
* Installing `ignition-cmake0` using a vcpkg before this PR is merged 
* Update to a vcpkg version  after this PR, and remove and reinstalled `ignition-cmake0` 

If a user is in that case, the easier workaround is to just remove the `downloads/temp` directory.

---

thanks for looking deeper in it. Your reasoning sounds good to me.